test/trento/hosts/events/host_registered_test.exs

```diff
@@ -15,11 +15,12 @@ defmodule Trento.Hosts.Events.HostRegisteredTest do
       os_version = Faker.App.version()
 
       assert %HostRegistered{
-               version: 4,
+               version: 5,
```

Comment:
why again did we need all this version updates again ?

Reply:
Because the event is upcasted